lib/cms/repo.ex

```diff
@@ -234,9 +234,11 @@ defmodule CMS.Repo do
   end
 
   defp params_to_string(params) when params == %{}, do: ""
+  defp params_to_string(nil), do: ""
+  defp params_to_string(params) when is_binary(params), do: params
 
   defp params_to_string(params) when is_map(params) do
-    [head | tail] = Enum.map(params, fn {k, v} -> "#{k}=#{v}" end)
+    [head | tail] = Enum.map(params, fn {k, v} -> "#{k}=#{params_to_string(v)}" end)
```

Comment:
We should not handle nested maps here as they don't really make sense. I would investigate what is calling this with a nested map. It is probably wrong.